test/index.test.ts

```diff
@@ -1,4 +1,3 @@
-
 // dummy test to add jest setup
 describe('test', () => {
```

Comment:
can we delete this? Is it needed?

Reply:
it is as this is where the index test is going for the first `buildDepTree()`. I'll tell you when `index.test.ts` and `index.ts` are ready for review. Those are last WIP part.